webdriver/tests/support/helpers.py

```diff
@@ -261,3 +272,26 @@ def find_new_handle(session):
 
     return wait.until(find_new_handle)
 
+
+def element_position(session, element):
+    """Returns the dimensions and location of the specified element."""
+    return dict(session.execute_script("""
+        return {
+            x: arguments[0].offsetLeft,
+            y: arguments[0].offsetTop,
+            width: arguments[0].clientWidth,
+            height: arguments[0].clientHeight,
+        };
+        """, args=(element,)))
+
+
+def scroll_to(session, x, y):
+    """Scroll to the specified location within the docuemnt."""
+    return session.execute_script("window.scrollTo({0}, {1});".format(x, y))
+
+
+def get_element_property(session, element, prop):
```

Comment:
I don't think that this is needed given that you can call `element.property(name)`, which the WebDriver client provides.